deploy.sh

```diff
@@ -0,0 +1,50 @@
+# Used for pushing to build repo in a travis-ci deploy
+set -e
+
+if [ "${TRAVIS_BRANCH}" == "master" ]; then DEPLOY_BRANCH="prod-beta"; fi
+if [ "${TRAVIS_BRANCH}" == "stable" ]; then DEPLOY_BRANCH="prod-stable"; fi
+if [ -z "${DEPLOY_BRANCH}" ]; then
+    echo "Not configured to deploy anything for this branch: ${TRAVIS_BRANCH}"
+    exit 1
+fi
+
+if [ -z "${GITHUB_AUTH}" ]; then
+    echo "Env var for github credentials not provided by Travis"
+    exit 1
+fi
+
+# Set up git and check out insights-frontend-build
+mkdir -p ~/insights-frontend-build
+cd ~/insights-frontend-build
+git config --global user.email "travis@travis-ci.org"
+git config --global user.name "Travis CI"
+git init .
+git remote add origin https://${GITHUB_AUTH}@github.com/RedHatInsights/insights-frontend-build.git
+git fetch origin
+git checkout -b ${DEPLOY_BRANCH}
+cd -
+
+# Get tag from package.json
+node --max_old_space_size=4096 --optimize_for_size --stack_size=4096 ./node_modules/.bin/gulp prod
+export PKG_TAG=$(node -e 'console.log(require("./package.json").version)')
+
+# Add some of our tags into the html content
+cd build
+sed -s "s|___TAG___|${PKG_TAG}|g" -i index.html
+sed -s "s|___COMMIT___|${TRAVIS_COMMIT}|g" -i index.html
+if [ "${DEPLOY_BRANCH}" == "prod-beta" ]; then
+    cp ./index.html ./indexbeta.html
```

Comment:
Does this get renamed back to index.html at some point? this part always confused me.

Reply:
Well, technically both index and indexbeta are present in the build dir since it's a `cp`... this is what was going on in @iphands build script, can't say I fully understand it either ;)